src/item.cpp

```diff
@@ -2493,16 +2493,11 @@ int item::bash_resist() const
         return resist;
     }
     if (item::item_tags.count("leather_padded") > 0){
-        l_padding = volume() * (float(get_coverage()) / 100) / 2.5;
-        if (l_padding > 5 ){
-            l_padding = l_padding / 2.5;   //Hard cap so coats don't become solid steel
-        }
+               //Max value           Steepness of graph         Center of 'S'
```

Comment:
Ok, makes sense.

> (which btw. seems strange, shouldn't Kevlar be better?)

Kevlar is currently about 2x better for cut_resist, but the same for bash. Looking at the cut and bash resist for leather armors and the kevlar vest, I was actually leaning toward making leather *better* at bash resist, but since kevlar is a fairly scarce resource, I decided to make it the same.